### ISSUE TITLE
Scrollbars in Firefox for Tabs (and maybe Wizard)

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-next#1103]
As of today, Firefox has increased support for scrollbars in terms of color and width. This means that it is possible for us to hide scrollbars across Chrome, Firefox and IE (need to check with Edge). 
This affects the Tabs component and Wizard component.
The scrollbar should be hidden in Chrome and its width set to none in Firefox aligning with what the original design had specified.

- https://drafts.csswg.org/css-scrollbars-1/  
- https://hacks.mozilla.org/2018/12/firefox-64-released/ 

Related issue #859

**Comment by matthewcarleton** [comment]
I've played around with this a bit and noticed that if I set in my OS to show scrollbars `scrollbar-width: none;` still hides them. 
I'm not sure this is the right approach. Hiding something the user expects to see because they set it isn't great for a11y. 
In webkit the browser only shows the scrollbar above the content and when it's active which is better. We can't force this in Firefox though :(
